DependencyInjection/Configuration.php

```diff
@@ -15,8 +15,14 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $treeBuilder->root('lexik_data_layer');
+        $treeBuilder = new TreeBuilder('lexik_data_layer');
+
+        if (method_exists($treeBuilder, 'getRootNode')) {
+            $rootNode = $treeBuilder->getRootNode();
```

Comment:
I kept the both ways of getting the root node here in case it is needed in the future.